systemtests/src/main/java/io/enmasse/systemtest/platform/Kubernetes.java

```diff
@@ -564,13 +568,21 @@ public void deletePod(String namespace, String podName) {
      * @throws Exception whe deployment failed
      */
     public void createDeploymentFromResource(String namespace, Deployment resources) throws Exception {
+        createDeploymentFromResource(namespace, resources, 2, TimeUnit.MINUTES, true);
+    }
+
+    public void createDeploymentFromResource(String namespace, Deployment resources, long time, TimeUnit unit, boolean doLog) throws Exception {
         if (!deploymentExists(namespace, resources.getMetadata().getName())) {
             Deployment depRes = client.apps().deployments().inNamespace(namespace).create(resources);
             Deployment result = client.apps().deployments().inNamespace(namespace)
-                    .withName(depRes.getMetadata().getName()).waitUntilReady(2, TimeUnit.MINUTES);
-            log.info("Deployment {} created", result.getMetadata().getName());
+                    .withName(depRes.getMetadata().getName()).waitUntilReady(time, unit);
+            if (doLog) {
```

Comment:
Is it really necessary to enable/disable logging per method invocation? Does this belong somewhere at the class level or Environment?

Reply:
with this changes I only disabled the logging in a few places where it was too verbose for the scale test, I'm not sure I want to disable logging for this full class, altough I agree this is not the most elegant solution